src/CheckoutSdk/Instruments/Four/InstrumentsClient.cs

```diff
@@ -52,5 +53,13 @@ public Task<object> Delete(string instrumentId, CancellationToken cancellationTo
                 SdkAuthorization(),
                 cancellationToken);
         }
+
+        public Task<BankAccountFieldResponse> GetBankAccountFieldFormatting(CountryCode country, Currency currency,
+            BankAccountFieldQuery bankAccountFieldQuery, CancellationToken cancellationToken = default)
+        {
+            CheckoutUtils.ValidateParams("country", country, "currency", currency, "bankAccountFieldQuery", bankAccountFieldQuery);
+            return ApiClient.Query<BankAccountFieldResponse>(BuildPath("validation/bank-accounts", country.ToString(), currency.ToString()),
+                SdkAuthorization(), bankAccountFieldQuery, cancellationToken);
```

Comment:
There is one thing missing here.
If you see the base call, we are sending `SdkAuthorizationType.SecretKeyOrOAuth` but this new endpoint is only available during a OAuth context. If a customer with secret keys tries to use this endpoint this will not work.
You need to overload the `sdkAuthorization` to only use `SdkAuthorizationType.OAUTH` like we did in Java.